test/parser.test.js

```diff
@@ -210,7 +210,7 @@ describe('parser', () => {
     i18nextParser.end(fakeFile)
   })
 
-  it('creates two files per namespace and per locale', (done) => {
+  it('creates one file per namespace and per locale', (done) => {
```

Comment:
Maybe I would add a test to check for `_old` catalog creation then. I saw the test below but the catalog already exist and it tests mostly mergeability of the old catalog.